src/layouts/AllServices.tsx

```
@@ -83,9 +83,9 @@ const AllServices = ({ Footer }: AllServicesProps) => {
     id: 'otherServices',
     title: 'Other Services',
     navItems: [
-      { id: 'redhatProductTrials', title: 'Red Hat Product Trials', isExternal: true },
-      { id: 'trustedArtifactSigner', title: 'Trusted Artifact Signer', isExternal: true },
-      { id: 'trustedAnalyzer', title: 'Trusted Profile Analyzer', isExternal: true },
+      { id: 'redhatProductTrials', title: 'Red Hat Product Trials', href: 'https://www.redhat.com/en/products/trials', isExternal: true },
+      { id: 'trustedArtifactSigner', title: 'Trusted Artifact Signer', href: 'https://developers.redhat.com/learn/install-sign-verify-using-red-hat-trusted-artifact-signer', isExternal: true },
```

ESLint

[error] Replace `·id:·'trustedArtifactSigner',·title:·'Trusted·Artifact·Signer',·href:·'https://developers.redhat.com/learn/install-sign-verify-using-red-hat-trusted-artifact-signer',·isExternal:·true` with `⏎········id:·'trustedArtifactSigner',⏎········title:·'Trusted·Artifact·Signer',⏎········href:·'https://developers.redhat.com/learn/install-sign-verify-using-red-hat-trusted-artifact-signer',⏎········isExternal:·true,⏎·····`

(prettier/prettier)